hotfix/2.5.1/delete-old-logs.patch

```diff
@@ -0,0 +1,10 @@
+--- /dev/null
++++ b/10_delete_old_logs.sh
+@@ -0,0 +1,7 @@
++#!/bin/sh
++set -e
++
++# Up to version 2.5.0 the logs of the contacts service were written into a
++#  file that was not picked up by logrotate.
```

Comment:
Nitpick, some extra space

```suggestion
+# file that was not picked up by logrotate.
```

Reply:
This extra space is already on master, so I'd rather keep it as is :)

_Also: It is intentional to denote the continuation of the sentence in the last line._